### PR TITLE
updated v2.0.3

### PR DESCRIPTION
**Pull request by 0xbe7a**
- **updated v2.0.2**
- **MNT: Re-rendered with conda-build 24.5.0**
- **update to 2.0.3**
- **fix maintainers**

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by 0xbe7a** [comment]
@conda-forge-admin, pleasee rerender

**Comment by 0xbe7a** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jonashaag) [inline recipe/meta.yaml:43]: lol

**Review by jonashaag — APPROVED** [review]

**Comment by 0xbe7a** [comment]
I can't merge this as 48762 is maintainer, not '0xbe7a'

**Pull request merged**
